src/VerticalDivider/verticaldivider.css

```diff
@@ -0,0 +1,13 @@
+
+@each $color $tone in $base-colors {
+    .u-vertical-divider--#{$tone} {
+      border-right: 1px solid $color;
+    }
+  }
+
+@each $value $space in $spacingMap {
```

Comment:
think you can remove this @each block right?

Reply:
ah yes good catch